src/test/java/com/saucelabs/bamboo/sod/action/BuildConfiguratorTest.java

```diff
@@ -29,10 +31,10 @@
 import java.util.List;
 import java.util.Map;
 
+import static com.saucelabs.bamboo.sod.config.SODKeys.TEMP_TUNNEL_ID;
```

Comment:
In my opionion, Static imports make reading code harder. Can you just use SODKeys.TEMP_TUNNEL_ID in your code?